### DISCUSSION TITLE
MCMC trace plot and corner plot for `DESIJ0201-2739`

### DISCUSSION THREAD

**Discussion by rafee-adnan** [AstroBridge/BDLensing#108] [Show and tell]
I ran mcmc for a total of `5507` steps. As we can see from the trace plot, most of the parameters converges eventually. I took the last `992` steps for plotting the corner plot. I am sharing my trace plot and corner plot. Kindly comment if my mcmc run lacks anything.

![Trace_plot_DESIJ0201-2739](https://github.com/ajshajib/BDLensing/assets/76589056/2a69ffb1-8001-42dd-b30b-71d8743b8e28)
![Corner_plot_DESIJ0201-2739](https://github.com/ajshajib/BDLensing/assets/76589056/ec1a2136-abab-412a-867a-d508da6e4c66)

**Comment by rafee-adnan** [comment]
Now I have 28 parameters to fit in the MCMC run. I am facing a problem. No matter how high I set the upper limit for `gamma_lens0`, it always increases. I have run this mcmc for about, 11000 steps and still no sign of convergence. Any suggestions will greatly help. I should mention that the reduced chi squared is decreasing and after 10872 steps the latest value is `0.9807`
![Trace_plot_DESIJ0201-2739 (1)](https://github.com/ajshajib/BDLensing/assets/76589056/1be5dbc0-d43d-474d-9695-065941eec701)
<img width="733" alt="Screenshot 2023-06-23 at 2 01 53 AM" src="https://github.com/ajshajib/BDLensing/assets/76589056/e879c5b6-8a5b-4296-a048-84edd33e8be4">

**Reply by rafee-adnan** [reply]
@ajshajib vai, is this situation normal? What should I do next?

**Comment by Rajit13** [comment]
I think @rafee-adnan vai's issue is solved, right?

**Reply by rafee-adnan** [reply]
Yeah, it's solved!
